src/main/java/com/datapipe/jenkins/vault/credentials/VaultGithubTokenCredential.java

```diff
@@ -16,12 +16,21 @@
     private final @NonNull
     Secret accessToken;
 
+    private final @NonNull
+    String mountPath;
+
     @DataBoundConstructor
     public VaultGithubTokenCredential(@CheckForNull CredentialsScope scope,
         @CheckForNull String id,
         @CheckForNull String description,
+        @NonNull String mountPath,
         @NonNull Secret accessToken) {
         super(scope, id, description);
+        if (mountPath.isEmpty()) {
+            this.mountPath = DescriptorImpl.defaultPath;
+        } else {
+            this.mountPath = mountPath;
+        }
         this.accessToken = accessToken;
     }
```

Comment:
Use databound setter instead of modifying the constructor.

Reply:
Also `isEmpty` does not consider whitespace or null.

https://github.com/jenkinsci/hashicorp-vault-plugin/blob/3d36af992d835a1267b09c2584cf9eeb6daf5e90/src/main/java/com/datapipe/jenkins/vault/credentials/VaultAppRoleCredential.java#L32-L33

Reply:
Thanks for feedback @jetersen. 

1. Added `DataBoundSetter` for mountPath.
2. Using `defaultIfBlank` to check for null/empty/whitespace. Added unit tests also for the same

Amended the changes to the same commit. Please review it.